### PR TITLE
Fixed various tests on MySQL with MyISAM storage engine.

### PR DESCRIPTION
**Pull request by felixxm**

### CODE REVIEW

File: django/db/backends/mysql/features.py [hunk 1]
```diff
@@ -305,6 +305,9 @@ def supports_transactions(self):
         """
         return self._mysql_storage_engine != "MyISAM"
 
+    uses_savepoints = property(operator.attrgetter("supports_transactions"))
```
Comment (hramezani) [inline django/db/backends/mysql/features.py:308]: As both features use support_transactions, can't we make a support_transactions feature?

Comment (hramezani) [inline django/db/backends/mysql/features.py:308]: As both features use support_transactions, can't we make a support_transactions feature?

Comment (hramezani) [reply django/db/backends/mysql/features.py:308]: Sorry. Can't we directly use `supports_transactions` feature flag?

File: django/db/backends/mysql/features.py [hunk 2]
```diff
@@ -305,6 +305,9 @@ def supports_transactions(self):
...
+    can_release_savepoints = property(operator.attrgetter("supports_transactions"))
```
Comment (hramezani) [inline django/db/backends/mysql/features.py:309]: I can't see any usage of `can_release_savepoints`.
Please ignore my comment if it's not related.

Comment (felixxm) [reply django/db/backends/mysql/features.py:309]: It's used in `transactions` tests.

**Review by hramezani — APPROVED** [review]
👍

**Comment by felixxm** [comment]
@hramezani Thanks :+1:

**Pull request merged**